pkg/chains/formats/slsa/README.md

```diff
@@ -12,4 +12,314 @@ Shown below is the mapping between Tekton chains proveance and SLSA predicate.
 |Tekton Chains Provenance Format version     | SLSA predicate | Notes |
 |:------------------------------------------|---------------:|------:|
 |**slsa/v1**| **slsa v0.2**  | same as currently supported `in-toto` format|
-|**slsa/v2alpha1**| **slsa v0.2**  | contains complete build instructions as in [TEP0122](https://github.com/tektoncd/community/pull/820). This is still a WIP and currently only available for taskrun level provenance. |
+|**slsa/v2alpha1**| **slsa v0.2**  | contains complete build instructions as in [TEP0122](https://github.com/tektoncd/community/pull/820) and the [example below](#provenance-format-example-for-slsa/v2alpha1). This is still a WIP and currently only available for taskrun level provenance. |
+
+## Provenance Format Example for slsa/v2alpha1
+When running the following taskrun with bundle resolver:
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  generateName: bundles-resolver-
+spec:
+  workspaces:
+    - name: output
+      emptyDir: {}
+  podTemplate:
+    securityContext:
+      fsGroup: 65532
+  taskRef:
+    resolver: bundles
+    params:
+      - name: bundle
+        value: gcr.io/tekton-releases/catalog/upstream/git-clone@sha256:8e2c3fb0f719d6463e950f3e44965aa314e69b800833e29e68ba2616bb82deeb
```

Comment:
nit: May be more informative to use a tag-style uri here in the spec in order to show that the actual corresponding sha was resolved and presented in the configsource?

i.e. gcr.io/tekton-releases/catalog/upstream/git-clone:0.8

Reply:
Sure. I can do that! Thanks

Reply:
Done!